src/completed.js

```diff
@@ -0,0 +1,34 @@
+const todoes = JSON.parse(localStorage.getItem('todoes'));
+
+const DeleteChecked = () => {
+  const temp = todoes.filter(todo => todo.completed === false);
+
+  localStorage.setItem('todoes', JSON.stringify(temp));
+  const newTodoes = JSON.parse(localStorage.getItem('todoes'));
+
+  for (let i = 0; i < newTodoes.length; i += 1) {
+    newTodoes[i].index = i + 1;
+  }
+  localStorage.setItem('todoes', JSON.stringify(newTodoes));
+  window.location.reload();
+};
+
+const True = (index) => {
+  for (let i = 0; i < todoes.length; i += 1) {
+    if (index === todoes[i].index) {
+      todoes[i].completed = true;
+      localStorage.setItem('todoes', JSON.stringify(todoes));
+    }
+  }
+};
+
+const False = (index) => {
+  for (let i = 0; i < todoes.length; i += 1) {
+    if (index === todoes[i].index) {
+      todoes[i].completed = false;
+      localStorage.setItem('todoes', JSON.stringify(todoes));
+    }
+  }
+};
```

Comment:
Great job implementing these functions for checking and unchecking the todos, however, you can fuse them on a single function that takes a bool and makes a different action according to its value.